Source/ARTLocalDevice.h

```diff
@@ -16,6 +16,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (instancetype)init NS_UNAVAILABLE;
 
+- (void)reset;
```

Comment:
This should probably be in the private interface.

Reply:
https://github.com/ably/ably-cocoa/pull/1259/commits/6c099ddb7b949d036e8585d95bb6278ff4d5e092